script.js

```diff
@@ -0,0 +1,30 @@
+function calculateAnswer () {
+    
+    let firstNumberElement = document.getElementById("firstNumber")
+    let secondNumberElement = document.getElementById("secondNumber")
+
+    let firstNumberValue = Number(firstNumberElement.value)
+    let secondNumberValue = Number(secondNumberElement.value)
+
+    let operationElement = document.getElementById("operation")
+    let operationSelected = operationElement.options[operationElement.selectedIndex].value
+
+    let answerElement = document.getElementById("answer")
+    answerElement.innerHTML = generateAnswer(firstNumberValue, secondNumberValue, operationSelected)
+}
+
+function generateAnswer (firstNumberValue, secondNumberValue, operationSelected) {
```

Comment:
Functions look great!! Just as a reminder, your arguments and parameters do not need to have the same names. It's fine if they do and can be less confusing that way, but if you wanted your function parameters to be called (op, num1, num2) for example, these parameters would still take on the values that are passed to them, in order, from where you call the function on like 13.